docs/api/ShallowWrapper/debug.md

```diff
@@ -40,14 +40,11 @@ Outputs to console:
 ```
 
 ```jsx
+const handleClick = () => {};
 const wrapper = shallow((
   <Book
     title="Huckleberry Finn"
-    cover={{
-      url: 'http://some.url/to/img.png',
-      width: 40,
-      height: 80,
-    }}
+    cover={handleClick}
```

Comment:
I don't think the change to this line makes sense, why would a prop named `cover` be a function?